components/gfx/font.rs

```diff
@@ -486,20 +486,23 @@ impl FontGroup {
     pub fn find_by_codepoint<S: FontSource>(
         &mut self,
         font_context: &FontContext<S>,
-        codepoint: char,
+        character: char,
```

Comment:
We have renamed `codepoint` to `character` in this PR, would it make sense to rename the function as well? maybe something `find_by_character`?

Reply:
Oh, great point. I've gone ahead and renamed these parameters`codepoint` and `next_codepoint`. It may make sens to rename this function later, but for now it's probably best to keep the same name.